CBOnlineIos/Presentation/Login.swift

```diff
@@ -0,0 +1,186 @@
+        countryCode.textColor = UIColor.darkGray
+        
+        // Example of customizing the textField input accessory view
+        let items = [
+            UIBarButtonItem(barButtonSystemItem: UIBarButtonItem.SystemItem.save, target: self, action: nil),
+            UIBarButtonItem(title: "Item 1", style: .plain, target: self, action: nil),
+            UIBarButtonItem(title: "Item 2", style: .plain, target: self, action: nil)
+        ]
+        countryCode.textFieldInputAccessoryView = getCustomTextFieldInputAccessoryView(with: items)
+        
+        // The placeholder is an example phone number of the selected country by default. You can add your own placeholder :
+        countryCode.hasPhoneNumberExample = false
+        countryCode.attributedPlaceholder = NSAttributedString(string:"Phone Number", attributes: [NSAttributedString.Key.foregroundColor: UIColor.darkGray])
+        
+        // Set the flag image with a region code
+        countryCode.setFlag(countryCode: .IN)
+        
+        
+        countryCode.center = view.center
+    }
+    private func getCustomTextFieldInputAccessoryView(with items: [UIBarButtonItem]) -> UIToolbar {
+        let toolbar: UIToolbar = UIToolbar()
+        
+        toolbar.barStyle = UIBarStyle.default
+        toolbar.items = items
+        toolbar.sizeToFit()
+        
+        return toolbar
+    }
+    
+    @objc func dismissCountries() {
+        listController.dismiss(animated: true, completion: nil)
+    }
+    //MARK: Proceed Button operations
+    // In this block of code we will work over prcoceed buttion operations
+    @IBAction func Button(_ sender: UIButton) {
+        if numberCheck == 0 {
+            alertControllerIsEmplty()
+        }
+        else{
+            performSegue(withIdentifier: "logToVfy", sender: self)
+        }
+    }
+    // Alert controller function
+    func alertControllerIsEmplty(){
+        let alertController = UIAlertController(title: "Warning", message: "Please enter correct Phone Number", preferredStyle: .alert)
+        let okAction = UIAlertAction(title: "OK",style: .cancel, handler:nil)
+        alertController.addAction(okAction)
+        self.present(alertController, animated: true)
+        
+    }
+    //MARK: Data Passs for next ViewController
+    // In this we are passing phone number and country Code from one ViewControler to another
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if (segue.identifier == "logToVfy") {
+            let vc = segue.destination as! OTPVerify
+            vc.vfyPhonenumber = self.phoneNumberEntered
+        }
+    }
+}
+
+//MARK: View Controller Extension
+extension Login: FPNTextFieldDelegate {
+    //Lets you know when the phone number is valid or not.
+    func fpnDidValidatePhoneNumber(textField: FPNTextField, isValid: Bool) {
+        textField.rightViewMode = .always
+        if isValid{
+            print(
```

Comment:
why do you need print ?

Reply:
This function is for entered phone number validation for the selected country.